english/experience/experience.md

```diff
@@ -0,0 +1,15 @@
+you close your eyes and gaze at the point between the eyebrows
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.